src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -6669,50 +7401,48 @@ public void BreakingChange_ReturnOutByRef()
 
             comp = CreateCompilation(source);
             comp.VerifyDiagnostics(
-                // (6,20): error CS8166: Cannot return a parameter by reference 't' because it is not a ref parameter
-                //         return ref t;
-                Diagnostic(ErrorCode.ERR_RefReturnParameter, "t").WithArguments("t").WithLocation(6, 20));
+                // (14,16): error CS8347: Cannot use a result of 'R1.R1(ref object)' in this context because it may expose variables referenced by parameter 'o' outside of their declaration scope
+                //         return new R1(ref o);
+                Diagnostic(ErrorCode.ERR_EscapeCall, "new R1(ref o)").WithArguments("R1.R1(ref object)", "o").WithLocation(14, 16),
+                // (14,27): error CS8168: Cannot return local 'o' by reference because it is not a ref local
+                //         return new R1(ref o);
+                Diagnostic(ErrorCode.ERR_RefReturnLocal, "o").WithArguments("o").WithLocation(14, 27),
+                // (18,16): error CS8347: Cannot use a result of 'R2.R2(in object)' in this context because it may expose variables referenced by parameter 'o' outside of their declaration scope
+                //         return new R2(new object());
+                Diagnostic(ErrorCode.ERR_EscapeCall, "new R2(new object())").WithArguments("R2.R2(in object)", "o").WithLocation(18, 16),
+                // (18,23): error CS8156: An expression cannot be used in this context because it may not be passed or returned by reference
+                //         return new R2(new object());
+                Diagnostic(ErrorCode.ERR_RefReturnLvalueExpected, "new object()").WithLocation(18, 23));
         }
 
-        // Breaking change in C#11: The rvalue from a method invocation that
-        // returns a ref struct is safe-to-escape from ... the ref-safe-to-escape of all ref arguments.
+        // Breaking change in C#11: A ref to ref struct argument is considered
+        // an unscoped reference when passed to an __arglist.
         [Fact]
-        public void BreakingChange_RefStructReturnFromRefArguments()
+        public void BreakingChange_RefToRefStructInArglist()
```

Comment:
Does this test duplicate `MethodArgumentsMustMatch_06`?

Reply:
Yes, it's a subset of `MethodArgumentsMustMatch_06` but I wanted to clearly identify the breaking changes with specific tests, and also have the breaking change tests match the breaking change documentation.